src/main/java/com/nextcloud/common/OkHttpMethodBase.kt

```diff
@@ -115,7 +119,7 @@ abstract class OkHttpMethodBase(
         return response?.header(name)
     }
 
-    fun getRequestHeader(name: String): String? {
```

Comment:
Having a header, having it set to `""` or not at all might make a difference for some clients.
If the client is consuming it in Kotlin, one can use `getRequestHeader("dodgy") :? ""` instead.

That's a big if. Java is probably less fun.

Reply:
Return of this function was not touched, so it always returns either header or "", therefore String? is unnecessary. 

If we want to make sure that it is a different handling if no header is found, then we need to return either null or something else.
As it is "our" implementation, I am fine with keeping it like that: either return header or empty (aka. not found/set).